csun-metalab/nomi
# NAM-446 -- Course selection format should include day of week and time instead of course number

# Description
Added additional information to course bar as shown in mock-up. Added extra styling to match the mock-up.
  
# Testing
1. Visit the students page and ensure the course selector bar has the Days and Time of the course.
2. Ensure there is no breaking on different screen resolutions.
3. Make sure the course selector bar is visually appealing and has proper contrast.
4. Look at the mock-up and make sure all the features in the selector bar was added https://drive.google.com/open?id=1GquGEL8auyOvhcMXxme5S2giTo8VlH-g.
